stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequestFactory.kt

```diff
@@ -85,13 +82,21 @@ open class AnalyticsRequestFactory(
 
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     companion object {
+        @Volatile
+        @VisibleForTesting
+        var sessionId: UUID = UUID.randomUUID()
```

Comment:
Nit, as we don’t seem to update it outside `regenerateSessionId`.

```suggestion
        val sessionId: UUID = UUID.randomUUID()
```

Reply:
Did you mean `private set`?

Reply:
Yes 🙈